res/raw-sw600dp/linphonerc_default

```diff
@@ -35,4 +35,21 @@ external_domains=Sorenson.com,ZVRS.com,CAAG.com,PurpleVRS.com,GlobalVRS.com,Conv
 
 [rtp]
 rtcp_xr_enabled=0
+ 
+[video_codec_0]
+mime=H264
+rate=90000
+enabled=1
+recv_fmtp=profile-level-id=42801F
```

Comment:
in ace android and ace ios, 720p works fine. we did the  packetization  parameter programmatically in the app.